test-suite/src/test/scala/org/scalajs/testsuite/javalib/ConcurrentHashMapTest.scala

```diff
@@ -0,0 +1,110 @@
+package org.scalajs.testsuite.javalib
+
+import language.implicitConversions
+
+import scala.language.implicitConversions
+
+import org.scalajs.jasminetest.JasmineTest
+
+object ConcurrentHashMapTest extends JasmineTest {
+
+  describe("java.util.concurrent.ConcurrentHashMap") {
+
+    it("should store strings") {
+      val chm = new java.util.concurrent.ConcurrentHashMap[String, String]()
+      assert ( chm != null )
+
+      expect(chm.size()).toEqual(0)
+      chm.put("ONE", "one")
+      expect(chm.size()).toEqual(1)
+      expect(chm.get("ONE")).toEqual("one")
+      chm.put("TWO", "two")
+      expect(chm.size()).toEqual(2)
+      expect(chm.get("TWO")).toEqual("two")
+    }
+
+    it("should store integers") {
+      val chm = new java.util.concurrent.ConcurrentHashMap[Int, Int]()
```

Comment:
You probably want to `import java.util.concurrent.ConcurrentHashMap` so that you don't have to repeat all of this.